docs/release-info/release-notes.md

```diff
@@ -57,50 +57,689 @@ For tips about how to write a good release note, see [Release notes](https://git
 
 This section contains important information about new and existing features.
 
+### Improved native queries
```

Comment:
IMO we should explicitly mention "Support grouping on complex columns and nested arrays"